lib/workflow/index.js

```diff
@@ -0,0 +1,6 @@
+const Workflow = require('./client');
+
+module.exports = settings => (req, res, next) => {
+  req.workflow = new Workflow(settings.workflow, req.user.access_token);
```

Comment:
It's probably makes sense to pass `req.user.profile.id` for the `changedBy` param at this point, rather than at the call-site.

Reply:
yeah, or just `req.user` as we also need the token

Reply:
And if we do that, then it doesn't make much sense to then pass the request in rather than specific parameters.

I'm still not really happy that the methods on the workflow client class take two sets of parameters as arguments, and the only reason for that not being one set of parameters is that some come from the inbound request.

Reply:
yeah, I'll bin the request, now everything is in a working state.